components/centraldashboard/public/components/resources/kubeflow-icons.js

```diff
@@ -4,6 +4,11 @@
 */
 import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
+import '@polymer/iron-icons/communication-icons.js';
```

Comment:
Do we need all of these new imports or only a subset of them?

Reply:
This is a good point but:

1. I wouldn't know how to import just specific icons from these files, I had tried but didn't find a way
2. Even if it was possible to import just specific icons, I would prefer to import these bundles so that people can then configure the sidebar with whatever icons they like (via ConfigMap) without having to change the source code

Reply:
Makes sense, lets keep them then